### DISCUSSION TITLE
How do I send custom data to a custom create view?

### DISCUSSION THREAD

**Discussion by paulpreibisch** [Laravel-Backpack/community-forum#742] [Q&A (Help)]
Hi there, 
How do I actually send custom data to a custom create view?

**Comment by paulpreibisch** [comment]
It looks like I can do this with $this->crud->set('unique_id',$myValue),

and then access it in blade using {{$crud->get('unique_id')}}

**Comment by jcastroa87** [comment]
Hi @paulpreibisch did you try to create a custom operation?

In any case when you want to send data to a view, this is the way:

```
return view('vendor.backpack.crud.moderate', $this->data)
```

$this->data is your array, all this content you can get easily using blade.

Cheers.

**Comment by jcastroa87** [comment]
Hi @paulpreibisch did you have a chance to check about it?

Please, let us know if you need anything else.

Meanwhile, i will close the issue, but feel free to re-open or create a new one if needed.
